Formula/quarantinecli.rb

```diff
@@ -0,0 +1,21 @@
+class Quarantinecli < Formula
+  desc "CLI Utility to quarantine and manage quarantined files on macOS"
+  homepage "https://github.com/Serena-io/quarantineCLI"
+  url "https://github.com/Serena-io/quarantineCLI/archive/refs/tags/1.0.0.tar.gz"
+  version "1.0.0"
+  sha256 "69ab41c71ed37c5b47e7dc5ed54bde28f15eaf38c81d804f768f74fb482a28e6"
+  license "MIT"
+
+  depends_on :macos
+  depends_on :xcode => "12.3"
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    bin.install ".build/release/quarantinecli"
+  end
+
+  test do
+    FileUtils.touch("foo")
+    system "#{bin}/quarantinecli", "--quarantine", "foo"
```

Comment:
```suggestion
    touch "foo"
    system bin/"quarantinecli", "--quarantine", "foo"
```
We don't need interpolation here.

Can we verify that this did something?

Reply:
Is there a way to get the output of a test? If so, yes